manifests/r/RoyalApps/RoyalTS/6.00.50805.0/RoyalApps.RoyalTS.locale.en-US.yaml

```
@@ -0,0 +1,18 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: RoyalApps.RoyalTS
+PackageVersion: 6.00.50805.0
+PackageLocale: en-US
+Publisher: Royal Apps GmbH
+PackageName: Royal TS
+PackageUrl: https://www.royalapps.com/ts/win/features
+License: Copyright 2021 Royal Apps GmbH
+LicenseUrl: https://support.royalapps.com/support/solutions/articles/17000074360
+ShortDescription: Using RDP, VNC, SSH based terminals, S/FTP or web-based interfaces
+Moniker: royalts
+Tags:
+- Remote Desktop
```

Best practice is to use all lower case. Referring to the 1.0.0 schema: [defaultLocale.md#tags](https://github.com/microsoft/winget-pkgs/blob/master/doc/manifest/schema/1.0.0/defaultLocale.md#tags)

> The best practice is to present these terms in all lower case with hyphens rather than spaces.

```suggestion
Tags:
- remote
- desktop
- remote-desktop
```